proposed/0012-safecoin-implementation/0012-safecoin-implementation.md

```diff
@@ -145,10 +145,52 @@ This process is outlined as:
      - The safecoin close group then send a receipt message to the wallet address to inform the user
        of a new minted safecoin allocated to them.
 
+## Safecoin Management
+
+Each safecoin is represented as a piece of data held by the group closest to it's ID. Safecoin data structure is defined as:
+'''rust
+ID: 64 bytes
+OWNER: 64 bytes
+'''
+The ID of a safecoin is 64Bytes long, with the most meaningful 32 bits being sequenced index starts from 0 to 4294967295, and the left over part to be fullfiled with all zeros or other pre-defined pattern (to allow coin division).
+The OWNER of a safecoin is the wallet address provided by the pmid_node as mentioned above.
+
+The Safecoin Management group can only approve a farming request when no such targeted safecoin data has been created before.
+
+When being asked to transfer the owevership, the request transcript must provide a valid signature that can be verified by the stored OWNER, which is actually a public-key. And the the owner will be updated to the new owner.
+
+When being asked to burn a coin, the request can only be from the current owner. The piece of data will then be removed.
```

Comment:
I think expend this a little. So the Burn / `Delete` is signed by the user and passed via the Client Managers who can increase the users storage space at this amount.